modules/overrides/config_legacy.go

```diff
@@ -149,6 +151,7 @@ func (l *LegacyOverrides) toNewLimits() Overrides {
 			MaxLocalTracesPerUser:  l.MaxLocalTracesPerUser,
 			MaxGlobalTracesPerUser: l.MaxGlobalTracesPerUser,
 			TenantShardSize:        l.IngestionTenantShardSize,
+			MaxAttributeBytes:      l.IngestionMaxAttributeBytes,
```

Comment:
do we have tests that lock this copying behavior down?

Reply:
i am adding a test to make sure the conversion between the legacy and new format is working. is that what you're looking for?

Reply:
yeah, also just kind of asking if we have these tests generally?